README.md

```diff
@@ -10,129 +10,42 @@ Make sure you have TypeScript and @typescript-eslint/parser installed, then inst
 npm i -D eslint-plugin-expect-type
 ```
 
-Please also make sure the following packages are also installed:
-
-```sh
-npm i -D eslint @typescript-eslint/eslint-plugin @typescript-eslint/parser
-```
+> See [typescript-eslint's Getting Started docs](https://typescript-eslint.io/docs) for how to run ESLint on TypeScript files.
 
 ## Usage
 
-Please add the following options to your `.eslintrc`
+Add the following options to your [ESLint configuration file](https://eslint.org/docs/latest/user-guide/configuring/configuration-files):
 
 ```json
 {
-  "extends": ["plugin:eslint-plugin-expect-type/recommended"],
-  "parser": "@typescript-eslint/parser",
-  "parserOptions": {
-    "project": "./tsconfig.json"
-  },
-  "plugins": ["eslint-plugin-expect-type"],
-  "root": true
+  "extends": ["plugin:expect-type/recommended"],
+  "plugins": ["eslint-plugin-expect-type"]
 }
 ```
 
-Rule severity could be configured as follows
+<!-- prettier-ignore -->
+<!-- begin auto-generated rules list -->
 
-```json
-{
-  "rules": {
-    "expect-type/expect": "error"
-  }
-}
-```
-
-To skip Snapshot update when eslint is run with `--fix` could be configured as follows:
+💼 Configurations enabled in.\
+✅ Set in the `recommended` configuration.\
+🔧 Automatically fixable by the [`--fix` CLI option](https://eslint.org/docs/user-guide/command-line-interface#--fix).\
+💭 Requires type information.
 
-```json
-{
-  "rules": {
-    "expect-type/expect": ["error", { "disableExpectTypeSnapshotFix": true }]
-  }
-}
-```
-
-**Note: Make sure to use `eslint --ext .js,.ts` since by [default](https://eslint.org/docs/user-guide/command-line-interface#--ext) `eslint` will only search for .js files.**
-
-## Adding $ExpectType and $ExpectError type assertions
```

Comment:
I like that `README.md` is autogenerated, but I do think it hurts discoverability for this rule that there aren't any examples of what it looks like in action on the GitHub home page.

Could we keep a few front-page examples outside of the autogenerated part?